airflow/jobs.py

```diff
@@ -1610,16 +1610,17 @@ def _execute_helper(self, processor_manager):
             loop_start_time = time.time()
 
             # Traverse the DAG directory for Python files containing DAGs
-            # periodically
-            elapsed_time_since_refresh = (timezone.utcnow() -
```

Comment:
nit: would be nice to keep the variable name for readability.

Reply:
@jgao54 does "nit" mean you want me to change it back? :-P

Reply:
sorry, nit == nit-picking, and yeah I think it helps with readability :)